Plots/XY/NCL_xy_35.py

```diff
@@ -0,0 +1,249 @@
+"""
+NCL_xy_35.py
+============
+This script illustrates the following concepts:
+   - Use of opacity resources
+     to control opacity of curves and markers in
+     XYPlots.
+   - How to set line opacity and marker opacity to 
+     different values using RGBA tuples
+   - Opacity is still functional in colorless plots
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_35.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/xy_35_lg.png
+"""
+
+###############################################################################
+# Import packages:
+
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import colors
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Create data:
+
+# Make array of x-values, 64 evenly spaced values between 0 and 1
+f = np.linspace(0., 1., 64)
+
+twopi = 2 * 3.141592654
+
+# Create three arrays of y-values to be plotted
+x = np.cos(f * 2*twopi)
+y = np.cos(f * 3*twopi)
+z = x * y
+
+###############################################################################
+# Plot:
+
+# Create figure with 6 axes
+fig, axes = plt.subplots(figsize=(7, 10), nrows=3, ncols=2)
+plt.subplots_adjust(wspace=0.3, hspace=.4)
+
+# Plot first graph
+ax = axes[0, 0]
+
+# Use geocat-viz function to add major and minor ticks on the x and y axes
+gvutil.add_major_minor_ticks(ax,
```

Comment:
You have a lot of code here that is very similar. You may be able to use a loop to define all of the axes, ticks, tick labels, etc then after the loop you can plot the lines with the different capacities and markers as well as make the different titles. I think it might be neater and reader friendly.